com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/EmissionUIBlock.cs

```diff
@@ -122,7 +120,7 @@ public override void OnGUI()
             }
         }
 
-        void UpdateEmissiveColorFromIntensityAndEmissiveColorLDR()
+        public static void UpdateEmissiveColorFromIntensityAndEmissiveColorLDR(MaterialEditor materialEditor, Material[] materials)
```

Comment:
missing API doc

Reply:
nice catch, I changed them to internal because they shouldn't be public anyway